tr.md

```diff
@@ -63,7 +63,7 @@ Report Number | Author(s) | Title | Venue
 [TUD-SERG-2016-022](https://repository.tudelft.nl/islandora/search/TUD-SERG-2016-022?collection=research) | Arie van Deursen, Maurício Aniche, Joop Aué, Rogier Slag, Michael de Jong, Alex Nederlof, Eric Bouwers | A Collaborative Approach to Teaching Software Architecture | [SIGCSE 2017](http://sigcse2017.sigcse.org/) 
 [TUD-SERG-2016-021](https://repository.tudelft.nl/islandora/search/TUD-SERG-2016-021?collection=research) | Marieke Huisman, Herbert Bos, Sjaak Brinkkemper, Arie van Deursen, Jan Friso Groote, Patricia Lago, Jaco van de Pol, and Eelco Visser | Software that meets its Intent | [ISoLA 2016](http://www.isola-conference.org/isola2016/) 
 [TUD-SERG-2016-020](https://repository.tudelft.nl/islandora/search/TUD-SERG-2016-020?collection=research) | Sebastiaan Oosterwaal, Arie van Deursen, Roberta Coelho, Anand Ashok Sawant, Alberto Bacchelli | Visualizing Code and Coverage Changes for Code Review | [FSE 2016](http://www.cs.ucdavis.edu/fse2016/) 
-[TUD-SERG-2016-019](https://repository.tudelft.nl/islandora/search/TUD-SERG-2016-019?collection=research) | Marco di Biase, Magiel Bruntink, Alberto Bacchelli | A security perspective on code review: The case of Chromium | [SCAM 2016](http://www.ieee-scam.org/2016/) 
+[TUD-SERG-2016-019](https://pure.tudelft.nl/portal/files/47563530/TUD_SERG_2016_019.pdf) | Marco di Biase, Magiel Bruntink, Alberto Bacchelli | A security perspective on code review: The case of Chromium | [SCAM 2016](http://www.ieee-scam.org/2016/) 
```

Comment:
It is less volatile to link to the enclosing pure entry, which then also offers the abstract, link to DOI, and other relevant meta info. For your report this is: https://pure.tudelft.nl/portal/en/publications/a-security-perspective-on-code-review(971dc100-f7f9-4f34-abef-d3ed7f02b57d).html

Reply:
Thanks for your suggestion, link updated.